feature-libs/product-configurator/rulebased/components/add-to-cart-button/configurator-add-to-cart-button.component.ts

```diff
@@ -168,6 +169,28 @@ export class ConfiguratorAddToCartButtonComponent implements OnInit, OnDestroy {
     );
   }
 
+  protected isQuoteCartActive(): Observable<boolean> {
+    return this.activeCartFacade
+      .getActive()
+      .pipe(map((cart) => cart.quoteCode !== undefined));
+  }
+
+  protected getTranslationKeyForAddToCart(
+    isAddToCart: boolean
+  ): Observable<string> {
+    let translationKey = of('configurator.addToCart.confirmation');
+    this.isQuoteCartActive().subscribe((isQuoteActive) => {
```

Comment:
Subscribe and usage of 'of' is not needed..please just map from one obs into another